sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/v2/state/StateDataSourceTransformWithStateSuite.scala

```diff
@@ -126,8 +126,7 @@ class SessionGroupsStatefulProcessorWithTTL extends
  * Test suite to verify integration of state data source reader with the transformWithState operator
  */
 class StateDataSourceTransformWithStateSuite extends StateStoreMetricsTest
-  with AlsoTestWithEncodingTypes
-  with AlsoTestWithChangelogCheckpointingEnabled {
```

Comment:
did you lose the `AlsoTestWithChangelogCheckpointingEnabled`? Does `AlsoTestWithRocksDBFeatures` contain that?

Reply:
Yeah, AlsoTestWithRocksDBFeatures does test that - not sure why this is showing up in the diff.